lib/browser/browser.ts

```diff
@@ -210,48 +208,55 @@ class OktaAuthBrowser extends OktaAuthBase implements OktaAuth, SignoutAPI {
     this.authStateManager = new AuthStateManager(this);
   }
 
+  // TODO: deprecate in 5.0
   signIn(opts) {
-    const loginWithCredential = (opts) => {
-      opts = clone(opts || {});
-      const _postToTransaction = (options?) => {
-        delete opts.sendFingerprint;
-        return postToTransaction(this, '/api/v1/authn', opts, options);
-      };
-      if (!opts.sendFingerprint) {
-        return _postToTransaction();
-      }
-      return this.fingerprint()
-      .then(function(fingerprint) {
-        return _postToTransaction({
-          headers: {
-            'X-Device-Fingerprint': fingerprint
-          }
-        });
-      });
-    };
+    deprecate('This method will be deprecated in v5.0, please use signInWithCredentials() instead.');
 
-    const loginWithRedirect = async (opts: SigninWithRedirectOptions = {}) => {
-      if(this._pending.handleLogin) { 
-        // Don't trigger second round
-        return;
-      }
-  
-      this._pending.handleLogin = true;
-      this.setFromUri(opts.fromUri);
-      try {
-        if (this.options.onAuthRequired) {
-          return await this.options.onAuthRequired(this);
-        }
-        return await this.loginRedirect(undefined, opts);
-      } finally {
-        this._pending.handleLogin = null;
-      }
+    opts = clone(opts || {});
+    const _postToTransaction = (options?) => {
+      delete opts.sendFingerprint;
+      return postToTransaction(this, '/api/v1/authn', opts, options);
     };
+    if (!opts.sendFingerprint) {
+      return _postToTransaction();
+    }
+    return this.fingerprint()
+    .then(function(fingerprint) {
+      return _postToTransaction({
+        headers: {
+          'X-Device-Fingerprint': fingerprint
+        }
+      });
+    });
+  }
 
-    if (isSignInWithCredentialsOptions(opts)) {
-      return loginWithCredential(opts);
-    } else {
-      return loginWithRedirect(opts);
+  /**
+   * Alias method of signIn()
+   */ 
+  signInWithCredentials(opts: SignInWithCredentialsOptions) {
```

Comment:
If we are deprecating it, lets do the alias the other way. `signInWithCredentials` is the real method and `signIn` is just a wrapper.